symplyphysics/laws/dynamics/potential_energy_from_deformation.py

```diff
@@ -1,26 +1,49 @@
+"""
+Potential energy from deformation
+=================================
+
+Spring accumulates energy while being deformated. This law is known as the *Hooke's law*.
+
+**Conditions:**
+
+#. The deformation is elastic (reversible).
+"""
+
 from sympy import (Eq, solve)
-from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+from symplyphysics import (units, Quantity, Symbol, validate_input,
     validate_output)
 
-# Description
-## Spring accumulates energy while being deformated. This law is known as Hooke's law.
-## Law: E = k * x**2 / 2, where
-## E is potential energy of deformated spring
-## k is elastic koefficient
-## x is deformation
+spring_energy = Symbol("spring_energy", units.energy)
+"""
+The potential energy of the spring.
 
-# Conditions.
-## Deformation is elactic (reversible).
+Symbol:
+    E
+"""
 
-spring_energy = Symbol("spring_energy", units.energy)
 elastic_koefficient = Symbol("elastic_koefficient", units.force / units.length)
+"""
+The elastic coefficient of the spring.
```

Comment:
Use same name for spring elastic coefficient over the code. There is another name in previous law.